assets/css/omniSelectStyle.css

```diff
@@ -0,0 +1,100 @@
+body.prevent-scroll {
```

Comment:
Please reformat the file to have newline between every sections after `}\n`

Reply:
Done